README.md

```diff
@@ -64,7 +64,7 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/222919864-f472138c-12a2-4a86-b5e0-d2f1d4fd964b.png)

**[Optional]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalize to make your pull request more descriptive and professional 😊

Reply:
Noted. Thanks for the advice.